__tests__/app.test.js

```diff
@@ -353,4 +353,132 @@ describe("All requests", () => {
                 });
         });
     });
+    describe("GET /api/articles (queries)", () => {
+        test("200 will returned and list of articles when all queries are provided", () => {
+            return request(app)
+                .get("/api/articles?topic=mitch&sort_by=title&order=asc")
+                .expect(200)
+                .then(({ body }) => {
+                    const articles = body.articles;
+                    expect(articles).toBeSortedBy("title", {
+                        descending: false,
+                    });
+                    expect(articles).toHaveLength(12);
+                    expect(articles.body).toBeUndefined();
+                    articles.forEach((article) => {
+                        expect(article).toHaveProperty("author", expect.any(String));
+                        expect(article).toHaveProperty("title", expect.any(String));
+                        expect(article).toHaveProperty("article_id", expect.any(Number));
+                        expect(article).toHaveProperty("topic", expect.any(String));
+                        expect(article).toHaveProperty("created_at", expect.any(String));
+                        expect(article).toHaveProperty("votes", expect.any(Number));
+                        expect(article).toHaveProperty("article_img_url", expect.any(String));
+                        expect(article).toHaveProperty("comments_count", expect.any(String));
+                    });
```

Comment:
You do not have to check all the articles come back with the right properties - you covered this in your initial testing for the GET 'articles' endpoint

Reply:
You could, however, check that everything comes back with a topic of 'mitch'